qualtran/bloqs/chemistry/thc.py

```diff
@@ -33,8 +31,10 @@
 from qualtran.bloqs.basic_gates import Hadamard, Ry, Toffoli, XGate
 from qualtran.bloqs.multi_control_multi_target_pauli import MultiControlPauli
 from qualtran.bloqs.on_each import OnEach
+from qualtran.bloqs.select_swap_qrom import SelectSwapQROM
```

Comment:
There's a CirqGateAsBloq in the body, is this problematic?

Reply:
updated to not use `CirqGateAsBloq`